src/Cache/FileCacheStorage.php

```diff
@@ -52,10 +52,6 @@ public function load(string $key, string $variableKey)
 		[,, $filePath] = $this->getFilePaths($key);
 
 		return (static function () use ($variableKey, $filePath) {
-			if (!is_file($filePath)) {
-				return null;
-			}
-
 			$cacheItem = @include $filePath;
```

Comment:
since we already swallow possible errors and validate the result we read just after that, we don't need to check beforehand whether `$filePath` actually is a file.